Content.Server/Medical/BiomassReclaimer/BiomassReclaimerComponent.cs

```diff
@@ -12,10 +12,16 @@ public sealed partial class BiomassReclaimerComponent : Component
         [ViewVariables]
         public float RandomMessTimer = 0f;
 
+        /// <summary>
+        /// Makes the reclaimer store non-integer biomass leftovers, since biomass can only stack to integer amounts.
+        /// </summary>
+        [ViewVariables, DataField]
+        public float BiomassAccumulator = 0f;
```

Comment:
why not just make CurrentExpectedYield a float and then floor the value before the SpawnMultipleFromMaterial (then substract after so only the leftovers are left, would also need to update when we set the value to not override the leftover)